pagopa/io-app
# chore: [IOBP-479] Add temporary missing methods error in method selection screen

## Short description
This PR adds the (temporary) missing methods error in methods selection screen

## List of changes proposed in this pull request
- Refactored `WalletPaymentPickMethodScreen` to allow the error to be displayed
- Added `WalletPaymentMissingMethodsError` component
- Added required locale keys

## How to test
With the `io-dev-api-server`, simulate an empty user wallet. Start a new payment from **Profile > New Wallet > Payment** and check if the error is displayed correctly and with working buttons.

## Preview

https://github.com/pagopa/io-app/assets/6160324/65f4d64f-6980-4d36-a82d-873ccf505b5e
